app/controllers/discover.js

```diff
@@ -121,6 +127,39 @@ export default ApplicationController.extend({
         });
     },
 
+    transformTypes(obj) {
+        if (typeof (obj) !== 'object') {
+            return obj;
+        }
+        var keys = Object.keys(obj);
```

Comment:
Use [`for ... in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) here